### PR TITLE
Added documentation for cross tenant cmdlets

### PR DESCRIPTION
**Pull request by PowershellScripts**
The cmdlets are available in SPO Management Shell v. 16.0.22810.0 and their usage is described in this official article: https://learn.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-onedrive-migration-step2?view=o365-worldwide

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Scenario
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md:75]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## SYNTAX
+
+```powershell
+Get-SPOCrossTenantCompatibilityStatus -PartnerCrossTenantHostUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet allows you to determine the compatibility with the partner tenant before cross-tenant migration. You must be a SharePoint Online Administrator or Global Administrator to run it.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:27]:
```suggestion
This cmdlet allows you to determine the compatibility with the partner tenant before cross-tenant migration. You must be a SharePoint Online administrator or a global administrator to run it.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## EXAMPLES
+
+### -----------------------EXAMPLE 1-----------------------------
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:31]:
```suggestion
### Example 1
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+```powershell
+Get-SPOCrossTenantCompatibilityStatus -PartnerCrossTenantHostUrl https://contoso-my.sharepoint.com
+```
+
+Gets the compatibility status with the partner tenant Contoso.
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:38]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## PARAMETERS
+
+### -PartnerCrossTenantHostUrl
+The cross tenant url of the partner tenant. The partner tenant can determine this for you by running `Get-SPOCrossTenantHostUrl` on each of the tenants.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:43]:
```suggestion
The cross-tenant URL of the partner tenant. The partner tenant can determine this for you by running `Get-SPOCrossTenantHostUrl` on each of the tenants.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+author: 
+ms.author: 
+ms.reviewer:
+---
+
+# Get-SPOCrossTenantHostUrl
+
+## SYNOPSIS
+
+Returns cross tenant host url.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md:17]:
```suggestion
Returns the cross-tenant host URL.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+## SYNTAX
+
+```powershell
+Get-SPOCrossTenantHostUrl  [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Use this cmdlet to retrieve the cross tenant host url. You must be a SharePoint Online administrator or Global Administrator to run the cmdlet.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md:27]:
```suggestion
Use this cmdlet to retrieve the cross-tenant host URL. You must be a SharePoint Online administrator or a global administrator to run the cmdlet.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+## EXAMPLES
+
+### ------------ Example 1 --------------------
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md:31]:
```suggestion
### Example 1
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+```powershell
+Get-SPOCrossTenantHostUrl
+```
+
+The command returns the cross tenant host url.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md:37]:
```suggestion
This command returns the cross-tenant host URL.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Set-SPOCrossTenantRelationship -Scenario MnA -PartnerRole Target -PartnerCrossTenantHostUrl https://contoso-my.sharepoint.com
+```
+
+This cmdlet sends a trust request to Contoso tenant.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md:37]:
```suggestion
This cmdlet sends a trust request to the Contoso tenant.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+## PARAMETERS
+
+### -PartnerCrossTenantHostUrl
+The cross tenant url of the partner tenant. The partner tenant can determine this for you by running `Get-SPOCrossTenantHostUrl` on each of the tenants.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md:42]:
```suggestion
The cross-tenant URL of the partner tenant. The partner tenant can determine this for you by running `Get-SPOCrossTenantHostUrl` on each of the tenants.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PartnerRole
+Roles of the partner tenant you're establishing trust with. Use *source* if partner tenant is the source of the OneDrive migrations, and *target* if the partner tenant is the Destination.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md:58]:
```suggestion
Roles of the partner tenant you're establishing trust with. Use *source* if the partner tenant is the source of the OneDrive migrations, and *target* if the partner tenant is the destination.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md (hunk 1 above)
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOCrossTenantRelationship.md:75]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@get-itips Gentle reminder.

**Comment by tiburd** [comment]
Seeking internal review of new topics

**Comment by PowershellScripts** [comment]
> Seeking internal review of new topics

@tiburd If it helps, the description of these cmdlets and parameters is already published in the documentation here: https://learn.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-onedrive-migration-step2?view=o365-worldwide#parameter-definitions

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md (hunk 2 above)
Comment (samkabue) [reply sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md]: "SharePoint Administrator or a Global Administrator" as these are the formal Microsoft Entra built-in role names

[https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference)

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md (hunk 7 above)
Comment (samkabue) [reply sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md]: "SharePoint Administrator or a Global Administrator"

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 13]
```diff
@@ -0,0 +1,63 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocrosstenantcompatibilitystatus
+applicable: SharePoint Online
+title: Get-SPOCrossTenantCompatibilityStatus
+schema: 2.0.0
+author:
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:8]:
```suggestion
author: trent-green
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 14]
```diff
@@ -0,0 +1,63 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocrosstenantcompatibilitystatus
+applicable: SharePoint Online
+title: Get-SPOCrossTenantCompatibilityStatus
+schema: 2.0.0
+author: 
+ms.author:
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:9]:
```suggestion
ms.author: trgreen
```

**Review by get-itips — CHANGES_REQUESTED** [review]
Please add the suggestion to the other articles too, thank you

**Comment by yogkumgit** [comment]
@PowershellScripts

**Review by get-itips — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+
+## SYNTAX
+
+```powershell
+Get-SPOCrossTenantCompatibilityStatus -PartnerCrossTenantHostUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet allows you to determine the compatibility with the partner tenant before cross-tenant migration. You must be a SharePoint Online administrator or a global administrator to run it.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:27]: >SharePoint Online administrator or a global administrator [](http://example.com/codeflow?start=123&length=57)

Here and other places: please use the official role names "SharePoint Administrator or a Global Administrator" as these are the formal Microsoft Entra built-in role names

[https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference)

**Review by samkabue — CHANGES_REQUESTED** [review]
:clock1:

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md (hunk 15 above)
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md:27]:
```suggestion
This cmdlet allows you to determine the compatibility with the partner tenant before cross-tenant migration. You must be a SharePoint Administrator or a Global Administrator to run it.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+## SYNTAX
+
+```powershell
+Get-SPOCrossTenantHostUrl  [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Use this cmdlet to retrieve the cross-tenant host URL. You must be a SharePoint Online administrator or a global administrator to run the cmdlet.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantHostUrl.md:27]:
```suggestion
Use this cmdlet to retrieve the cross-tenant host URL. You must be a SharePoint Administrator or a Global Administrator to run the cmdlet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, my apologies. Please commit the suggestions in accordance with @samkabue's request. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by samkabue — APPROVED** [review]
:shipit:

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
Thank you, everyone!

**Pull request merged**